sdk/search/azure-search/src/main/java/com/azure/search/SearchApiKeyPipelinePolicy.java

```diff
@@ -25,14 +26,13 @@
      * @throws IllegalArgumentException when the api key is an empty string
      */
     public SearchApiKeyPipelinePolicy(SearchApiKeyCredential apiKey) {
-        Objects.requireNonNull(apiKey);
+        Objects.requireNonNull(apiKey, "'apiKey' cannot be null.");
```

Comment:
Not necessary to have the message.
Consistent with other class
this.apiKey = Objects.requireNonNull(apiKey);

Reply:
It's been a general guideline to include a message on `Objects.requireNonNull` calls.